extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/things/controller.things.view.js

```diff
@@ -88,11 +145,13 @@ angular.module('PaperUI.things') //
             suggestedLabel : channel.label,
             suggestedCategory : channelType && channelType.category ? channelType.category : '',
             preSelectCreate : preSelect,
-            allowNewItemCreation : $scope.advancedMode && channel.kind !== 'TRIGGER' // allow "Create new Item" in
-            // advanced mode only, disable
-        // for normalMode or trigger
-        // channel
+            // allow "Create new Item" in advanced mode only, disable for normalMode or trigger channel (if
+            // not bound to a trigger profile)
```

Comment:
this comment is outdated. We allow item creation also for trigger channels in case the profile is a trigger profile.